exporter/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM gcr.io/google-appengine/debian9
+
+
+# Get the list of packages from the base image.
+ENV EXPORTER_VERSION 1.1.0
+ENV EXPORTER_SHA1 7261fba6fcb927b85de6fdc2576813b5a97f976f
+
+# Install Dependencies
+RUN set -x && \
+    apt-get update -y && \
+    apt-get install -y --no-install-recommends \
+    curl && \
+    govendor
```

Comment:
Please remove results off apt-get update at this step

Reply:
Removed.